src/main.ts

```diff
@@ -193,6 +194,8 @@ class TestReporter {
     core.info(`Check run create response: ${resp.status}`)
     core.info(`Check run URL: ${resp.data.url}`)
     core.info(`Check run HTML: ${resp.data.html_url}`)
+    core.setOutput('url', resp.data.url)
+    core.setOutput('url_html', resp.data.html_url)
```

Comment:
@peterwhitesell I've just noticed this is executed in a loop. Multiple check runs might be created if the input path has a pattern matching more files. In that case, the `url` variable would be set to the last one. What's your use case? We could leave it as is or output JSON encoded array instead. Not sure what's a better approach.